### PR TITLE
CORDA-3696: Updates to OS to enable JDK11 support

### PR DESCRIPTION
**Pull request by adelel1**
CORDA OS JDK 11 Testing branch.

This PR is now ready for merge in. It contains some updates to support JDK11. I have reverted the build system changes to build under JDK8 so now its currently running under JDK8 and ready for merge.

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/internal/NodeStartup.kt [hunk 1]
```diff
@@ -95,7 +98,19 @@ open class NodeStartupCli : CordaCliWrapper("corda", "Runs a Corda Node") {
...
     }
 
+    private fun disableJDK11Warnings() {
+        val theUnsafe = Unsafe::class.java.getDeclaredField("theUnsafe")
+        theUnsafe.setAccessible(true)
+        val unsafeObj = theUnsafe.get(null) as Unsafe
+        val loggerCls = Class.forName("jdk.internal.module.IllegalAccessLogger")
+        val logger = loggerCls.getDeclaredField("logger")
+        unsafeObj.putObjectVolatile(loggerCls, unsafeObj.staticFieldOffset(logger), null)
+    }
```
Comment (chrisr3) [inline node/src/main/kotlin/net/corda/node/internal/NodeStartup.kt]: Isn't there a `java.util.logging` way of disabling this warning? Using a banned class like `sun.misc.Unsafe` to disable a warning that only appears on Java 11 seems like an own goal.

OK, I see that `IllegalAccessLogger` is _not_ a `java.util.logging` Logger after all :cry: .

**Comment by chrisr3** [comment]
run tests: slow

**Review by chrisr3 — COMMENTED** [review]
It's worth checking is this "logger" field is a `java.uti.logging` Logger.

**Comment by cedricwalter** [comment]
@adelel1 I would expect that you also update quasar to 0.8 from 0.7
from https://github.com/puniverse/quasar/releases

- Supports JDK 11
- Dropped support of JDK 7 and 8

Maybe Corda use a forked version of quasar?

**Review by mnesbit — APPROVED** [review]
OK

**Pull request merged**